extractor/src/main/resources/projects-for-example.yml

```diff
@@ -4,6 +4,44 @@ SummaryJavadoc:
       - WxJava
       - openjdk17
 
+RequireThis:
+  Example1:
+    projects:
+      - default
```

Comment:
What is default?
Better to reference there all projects

Reply:
Left default, now referencing all projects. Done.